test_scripts/Resumption/InteriorVehicleData/001_InteriorVD_resumption_one_moduleType_without_moduleId_unexpected_disconnect.lua

```diff
@@ -13,7 +13,7 @@
 -- 1. Transport disconnect and reconnect are performed
 -- 2. App starts registration with actual hashId after unexpected disconnect
 -- SDL does:
--- - a. send RC.GetInteriorVD(subscribe=true, module_1, default moduleId) to HMI during resumption data
+-- - a. send RC.GetInteriorVehicleData(subscribe=true, module_1, default_moduleId) to HMI during resumption data
```

Comment:
Please replace `GetInteriorVD` to `GetInteriorVehicleData` in description in 012, 024

Reply:
Done in 024 (see 3f22932 )